File: Assignments/Assignment_5/program3.c
// Check Leap Year

#include<stdio.h>

////////////////////////////////////////////////////////////////////
//
//  Function Name : CheckLeapYear
//  Description :   check leap year
//  Input :         Integer
//  Output :        Integer
//  Author :        Omkar Sachin Narlawar
//  Date :          26/10/2025
//
////////////////////////////////////////////////////////////////////

void CheckLeapYear(int Year)
{
    if(Year % 400 == 0)
    {
        ("%d is Leap Year",Year);
    }
    else if(Year % 100 == 0)
    {
        printf("%d is not Leap Year",Year);
    }
    else if(Year % 4 == 0)
    {
        printf("%d is Leap Year",Year);
    }
    else
    {
        printf("%d is not Leap Year",Year);
    }
}

int main()
{
    int yr = 0;

    printf("Enter Year\n");
    scanf("%d",&yr);

    CheckLeapYear(yr);

    return 0;
}

File: Assignments/Assignment_28/Java_Programs/program3.java
/* Accept number of rows and columns from user and display below pattern.
 Input : iRow = 4     iCol = 4
 OUTPUT :
            A   A   A   A   A
            B   B   B   B   B
            C   C   C   C   C
*/

////////////////////////////////////////////////////////////////////////
//
//  Required Import File
//
////////////////////////////////////////////////////////////////////////

import java.util.*;

////////////////////////////////////////////////////////////////////////
//
//  Function Name   :   Display
//  Description     :   used to print patterns
//  Input           :   Integer
//  Output          :   Pattern
//  Author          :   Omkar Sachin Narlawar
//  Date            :   22/11/2025
//
////////////////////////////////////////////////////////////////////////

class Pattern
{
    public void Display(int iRow,int iCol)
    {
        int i = 0, j = 0;
        char ch = '\0';  

        ch = 'A';      

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++)
            {
                System.out.print(ch+"\t");
            }
            System.out.println();
            ch++;
        }
    }
}// End of Class

////////////////////////////////////////////////////////////////////////
//
//  Entry point function : Main
//
////////////////////////////////////////////////////////////////////////

class program3
{
    public static void main(String A[])
    {
        int iValue1 = 0, iValue2 = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of Rows : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter number of Columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Display(iValue1,iValue2);
    }
}// End of Main